python/pyspark/sql/pandas/serializers.py

```diff
@@ -454,6 +474,74 @@ def __repr__(self):
         return "ArrowStreamPandasUDFSerializer"
 
 
+class ArrowStreamPandasUDTFSerializer(ArrowStreamPandasUDFSerializer):
```

Comment:
I created a new serializer, rather than modifying the PandasUDFSerializer, to avoid any risk of inadvertently changing the behavior of pandas UDFs. cc @HyukjinKwon @ueshin @xinrong-meng